test/testapp.go

```diff
@@ -1,4 +1,4 @@
-// Package test provides utility drivers for running UI tests without rendering
+// Package test provides utility drivers for running UI tests without and without rendering.
```

Comment:
> // Package test provides utility drivers for running UI tests without and without rendering.

Duplicate "without"

```suggestion
// Package test provides utility drivers for running UI tests without rendering to a screen.
```

Reply:
A bit late of a comment, but thanks. My intent was to say "with or without" rendering, but your suggestion was even better :)

Reply:
"A bit late of a comment"?

Reply:
I meant that I committed your change early this morning but sent the comment many hours later.